commerce_coordinator/apps/commercetools/clients.py

```diff
@@ -268,3 +270,37 @@ def get_product_variant_by_course_run(self, cr_id: str) -> Optional[CTProductVar
             return None
 
         return matching_variant_list[0]
+
+    def create_return_for_order(self, order_id: str, order_version: str, order_line_id: str) -> CTOrder:
+        """
+        Creates refund/return for Commercetools order
+        Args:
+            order_id (str): Order ID (UUID)
+            order_version (str): Current version of order
+            order_line_id (str): ID of order line item
+        Returns (CTOrder): Updated order object or
+        Returns Exception: Error if update was unsuccesful.
+        """
+
+        try:
+            return_item_draft = ReturnItemDraft(
+                quantity=1,
+                line_item_id=order_line_id,
+                shipment_state=ReturnShipmentState.RETURNED
```

Comment:
Question: Should we also set the [ReturnPaymentState](https://docs.commercetools.com/api/projects/orders#returnpaymentstate) to `Initial`?  Since it is not set by default (not mentioned in their docs).

Reply:
Setting the ReturnShipmentState to `Returned` automatically sets ReturnPaymentState to `Initial`. It's in their docs and Phil and I tested it as well. Can see the example in the mocking of Add ReturnInfo thread Phil posted team-sonic-eng.